### ISSUE TITLE
No Streaming Response for Application Load Balancer + Lambda (Fastapi : server sent event -> streaming response) 

### ISSUE THREAD

**Issue by GxWwT** [Kludex/mangum#341]
if chat_request.stream:
        return StreamingResponse(
            content=model.chat_stream(chat_request), media_type="text/event-stream"
        )

**This approach, when redirected to ALB through Mangum, loses the streaming response. Instead, all chunks are printed out all at once.**

**Comment by KamilLegault** [comment]
I dont think this is a mangum issue, the Python runtimes on AWS Lambda do not support streaming.

Look into the AWS lambda web adapter https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/fastapi-response-streaming

**Comment by Dawn-Of-Justice** [comment]
Any update on this? I am also facing the same issue. The streaming works without mangum.
